nncf/torch/quantization/default_quantization.py

```diff
@@ -62,11 +62,27 @@
         operator_metatypes.PTDeformConv2dMetatype,
         operator_metatypes.PTModuleDeformConv2dMetatype,
         UnknownMetatype,
-        # Ticket: 108478
-        operator_metatypes.PTAbsMetatype,
-        operator_metatypes.PTExpMetatype,
-        operator_metatypes.PTLogMetatype,
-        operator_metatypes.PTSqrtMetatype,
+    ],
+    QuantizationTrait.QUANTIZATION_AGNOSTIC: [
```

Comment:
Looks like #1675 was merged without my approval. ReLU is a quantization-agnostic operation, in a sense that it doesn't matter if the quantization operation is placed immediately before or immediately after it, and should be marked as such everywhere. 

NNCF is right to propagate FQ through RELU if the RELU operation in question does not belong to any pattern. If you encounter models where this is an issue, analyze the model for a pattern and add the pattern.

Reply:
Please add a comment in all backends near this list stating just why RELU specifically is not considered to be QUANTIZATION_AGNOSTIC, namely:
1) that it's frequently better for the end accuracy to have quantizers set up after the RELU so that the input distribution to the quantizer is non-negative and we can therefore have better quantization resolution while preserving the original dynamic range, and:
2) that the runtime currently does not provide performance benefits from quantizing stand-alone RELU's, and maybe even on the contrary, and add a ticket reference.